package.json

```diff
@@ -54,6 +54,7 @@
     "moment": "2.18.1",
     "read": "1.0.7",
     "request": "2.81.0",
+    "requireg": "^0.1.6",
```

Comment:
I was so confused with this comment, thought you meant to remove the whole line 😛 I wonder if there is a setting to change npm to save it differently.